app/services/subjects/financial_incentive_set_subject_knowledge_enhancement_course_available_service.rb

```diff
@@ -9,7 +9,7 @@ def initialize(year:, financial_incentive: FinancialIncentive)
 
     def subject_with_subject_knowledge_enhancement_course_available
       subject_with_subject_knowledge_enhancement_course_available = {
-        2020 => ['Primary with mathematics', 'Biology', 'Computing', 'English', 'Design and technology', 'Geography', 'Chemistry', 'Mathematics', 'Physics', 'French', 'German', 'Spanish', 'Italian', 'Japanese', 'Mandarin', 'Russian', 'Modern languages (other)', 'Religious education']
+        2023 => ['Computing', 'Geography', 'Chemistry', 'Mathematics', 'Physics', 'French', 'German', 'Spanish', 'Italian', 'Japanese', 'Mandarin', 'Russian', 'Modern languages (other)']
```

Comment:
First time I see this code in Publish. Not related to your PR but does this code means that on 2024 cycle this won't work? I am intrigued by the existing code 🤔

Reply:
This is a good question. From what I can see, this class is only used in the seeds. Which means it really hasn't been doing anything since 2020. There is perhaps an argument for deleting it entirely. or for making it agnostic about the year. It is not something that impacts users. I've updated it for completeness rather than functionality. What do you think?

Reply:
Let's pair on these?

Reply:
Just summarising here so we don't wonder about it later. 
- The `subject_knowledge_enhancement_course_available` field in FinancialIncentives is not used in production -- it is false everywhere. 
- I have created a ticket to remove it so it does not confuse us again later. https://trello.com/c/3vseuZV2